### DISCUSSION TITLE
Help using ancombc2

### DISCUSSION THREAD

**Discussion by mradz19** [FrederickHuangLin/ANCOMBC#221] [General]
I am trying to use ancombc2 on my own dataset. I want to look for differential abundant taxa taxa between my three patient groups while factoring for other covariates such as age, BMI and antibiotic use.

My samples are split into three categorical groups (pathology) and each sample has additional metadata variables.

Would something like this be suitable?

```
ancombc2(phylo_object, assay_name="counts, tax_level="genus", fix_formular="Pathology+BMI+Antibiotics", group="Pathology", pairwise=T, global=T
```

Would that be suitable for identifying DAGS between pathology groups while factoring for BMI and antibiotic covariables?

**Comment by FrederickHuangLin** [comment]
Hi @mradz19,

Based on your description, I believe your code is correct.

Best,
Huang
